cmd/kube-apiserver/app/server.go

```diff
@@ -51,7 +51,8 @@ import (
 const (
 	// Maximum duration before timing out read/write requests
 	// Set to a value larger than the timeouts in each watch server.
-	ReadWriteTimeout = time.Minute * 60
+	ReadWriteTimeout            = time.Minute * 60
+	defaultLongRunningRequestRE = "(/watch(/.*|)$|/proxy(/.*|)$|logs(/|)$|portforward(/|)$|exec(/|)$)"
```

Comment:
I'm not sure.. @csrwng, are all params to pods/logs, pods/portforward, pods/exec done as query params, or can you also have path params?